blockchain/state/statedb.go

```diff
@@ -1111,6 +1111,9 @@ func (s *StateDB) PrepareAccessList(sender common.Address, feepayer common.Addre
 	//		s.AddSlotToAccessList(el.Address, key)
 	//	}
 	//}
+	if rules.IsMantle {
+		s.AddAddressToAccessList(coinbase)
```

Comment:
Since the function `AddAddressToAccessList` already performs a duplication check internally, it seems to appear unnecessary to include duplicate checks in both L1115 and L1096 (specifically, the condition `sender != feepayer`). This modification may be done in this PR together.

Reply:
Yes.

>  it's a pre-check


When it comes to EVM execution, prioritizing the removal of unnecessary executions seems crucial for refactoring. The dedicated PR you mentioned appears to be a better approach.

> L1115 seems irrelevant.

I considered that the coinbase may be a sender.

Reply:
@hyunsooda I have edited the comment, please check it. 
Applied at here, and please take a look. https://github.com/klaytn/klaytn/pull/1883/commits/9c31079c42a15937305ee35850b1e26964069036